src/fides/data/sample_project/docker-compose.yml

```diff
@@ -61,7 +61,7 @@ services:
     volumes:
       - type: bind
         source: ./privacy_center/config
-        target: /app/config
+        target: /fides/clients/privacy-center/config
```

Comment:
Note to self: we're pretty sure this doesn't work right now.

Reply:
...and I'd like to preserve this option to mount to `/app/config` for backwards-compat as per our docs: https://docs.ethyca.com/fides/dsr_quickstart/privacy_center

Reply:
Let's revert this change for now and keep it with `/app/config`.

Reply:
Reverting this causes the following error upon `"fides_env(test)"`:

```
2023-04-21 17:20:35 > start
2023-04-21 17:20:35 > next start
2023-04-21 17:20:35 
2023-04-21 17:20:36 ready - started server on 0.0.0.0:3000, url: http://localhost:3000
2023-04-21 17:20:37 error - Failed to load next.config.js, see more info here https://nextjs.org/docs/messages/next-config-error
2023-04-21 17:20:37 Error: Cannot find module '../config/config.json'
2023-04-21 17:20:37 Require stack:
2023-04-21 17:20:37 - /fides/clients/privacy-center/scripts/validate-config.js
2023-04-21 17:20:37 - /fides/clients/privacy-center/next.config.js
2023-04-21 17:20:37     at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1026:15)
2023-04-21 17:20:37     at Function.mod._resolveFilename (/fides/clients/privacy-center/node_modules/next/dist/build/webpack/require-hook.js:27:32)
2023-04-21 17:20:37     at Function.mod._resolveFilename (/fides/clients/privacy-center/node_modules/next/dist/build/webpack/require-hook.js:27:32)
2023-04-21 17:20:37     at Function.Module._load (node:internal/modules/cjs/loader:871:27)
2023-04-21 17:20:37     at Module.require (node:internal/modules/cjs/loader:1098:19)
2023-04-21 17:20:37     at require (node:internal/modules/cjs/helpers:108:18)
2023-04-21 17:20:37     at validateConfig (/fides/clients/privacy-center/scripts/validate-config.js:31:31)
2023-04-21 17:20:37     at Object.<anonymous> (/fides/clients/privacy-center/next.config.js:4:1)
2023-04-21 17:20:37     at Module._compile (node:internal/modules/cjs/loader:1196:14)
2023-04-21 17:20:37     at Object.Module._extensions..js (node:internal/modules/cjs/loader:1250:10) {
2023-04-21 17:20:37   code: 'MODULE_NOT_FOUND',
2023-04-21 17:20:37   requireStack: [
2023-04-21 17:20:37     '/fides/clients/privacy-center/scripts/validate-config.js',
2023-04-21 17:20:37     '/fides/clients/privacy-center/next.config.js'
2023-04-21 17:20:37   ]
2023-04-21 17:20:37 }
```

Can we update this once we fix the ability to load in config at runtime?

Reply:
Since I merged in https://github.com/ethyca/fides/pull/3135, this is addressed 👍